File: javascript-closure-angular-client/API/Client/WithholdDef.js
goog.provide('API.Client.WithholdDef');

/**
 * @record
 */
API.Client.WithholdDef = function() {}

/**
 * UUID Reference to an item in the LegalReason store.
 * @type {!string}
 * @export
 */
API.Client.WithholdDef.prototype.exemptionReasonCode;

/**
 * @type {!string}
 * @export
 */
API.Client.WithholdDef.prototype.customExemptionReasonDescription;

File: async-scala-client/src/main/scala/io/swagger/client/model/WithholdDef.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class WithholdDef (
  exemptionReasonCode: Option[String],  // UUID Reference to an item in the LegalReason store.
customExemptionReasonDescription: Option[String]
)
